.drone.yml

```diff
@@ -5840,6 +5840,148 @@ steps:
       ref:
       - refs/pull/*/head
 
+---
+##############################################################################################################
+# This pipeline bootstraps a tezedge node to 1000 and measures time this takes
+##############################################################################################################
+kind: pipeline
+name: perf/bootstrap-time-009-florence
+
+depends_on:
+  - build-tezedge-binaries
+
+node:
+  tag: real-time-linux-runner
+
+trigger:
+  branch:
+    - master
+    - develop
+  ref:
+    - refs/pull/**
+
+volumes:
+  - name: build
+    host:
+      path: /usr/local/etc/tezedge-ci/build/build_${DRONE_BUILD_NUMBER}/build_files
+  - name: identities
+    host:
+      path: /usr/local/etc/tezedge-ci/build/build_${DRONE_BUILD_NUMBER}/build_files/identities
+  - name: snapshots
+    host:
+      path: /usr/local/etc/tezedge-ci/data/
+  - name: tools
+    host:
+      path: /usr/local/etc/tezedge-ci/tools
+  - name: cache
+    host:
+      path: /usr/local/etc/tezedge-ci/data/cache/build_${DRONE_BUILD_NUMBER}/${DRONE_STAGE_NAME}
+
+steps:
+
+  - name: octez-node
+    image: tezos/tezos:v9-release
+    detach: true
+    environment:
+      TEZOS_NODE_DIR: /home/tezos/data
+    volumes:
+      - name: identities
+        path: /identities
+      - name: snapshots
+        path: /snapshots
+    commands:
+      - tezos-node config init --network florencenet
+      - cp /identities/identity_1.json $${TEZOS_NODE_DIR}/identity.json
+      - tezos-node snapshot import /snapshots/octez-009-florence-1000.snapshot --block BKqNZHy5CFYLD6v6BjDLrEavrkK4qH39hhrsD7QtbWcdnnwoYub
+      - tezos-node run --no-bootstrap-peers
+
+  - name: tezedge-new-node
+    image: tezedge/tezedge-ci-builder
+    user: root
+    volumes:
+      - name: build
+        path: /build
+      - name: tools
+        path: /tools
+    commands:
+      # let octez node to be up and running
+      - sleep 15
```

Comment:
@akoptelov 
you dont need this sleep, it is not accurate,
we have a step for that:
```
  - name: wait-octez-node-to-start
    image: tezedge/tezos-node-bootstrap:latest
    pull: if-not-exists
    commands:
      - tezos-node-bootstrap bootstrap --level=0 --nodes $${OCTEZ_NODE_RPC_CONTEXT_ROOT}
```